src/lib/ConfettiExplosion.svelte

```diff
@@ -20,7 +20,12 @@
 	const FLOOR_WIDTH = 1600; // horizontal spread of particles in pixels
 	const PARTICLE_COUNT = 150;
 	const DURATION = 3500;
-	const COLORS = ['#FFC700', '#FF0000', '#2E3191', '#41BBC7'];
+	const COLORS = ['#FFC700', '#F00', '#2E3191', '#41BBC7'];
+
+	const abs = Math.abs,
+		random = Math.random,
+		mathRound = Math.round,
+		max = Math.max;
```

Comment:
Bundles cannot minify names of global objects. By aliasing the function names they can be minified (ie the bundle will use a short function name like `a` instead of `Math.round`